### PR TITLE
Feature/100javaprograms venkatesh

### PR DESCRIPTION
**Pull request by AroonSachin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,14 +23,20 @@
...
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>4.3.0</version>
+			<version>4.6.0</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/io.appium/java-client -->
+		<dependency>
+			<groupId>io.appium</groupId>
+			<artifactId>java-client</artifactId>
+			<version>8.2.0</version>
```
Comment (AroonSachin) [inline pom.xml:32]: Please update the java-Client version to 8.3.0

File: src/main/java/commonuseractions/CommonActionMethods.java [hunk 2]
```diff
@@ -630,7 +649,174 @@ public String currentDate(int plusdays, String format) {
...
+					break;
+				}
+			} else if (isElementPresent(element) == true) {
+				if (sendkey) {
+					sendKeysMethod(element, keytoSend);
+				}
+				logMessage("Element is in view ");
+				break;
+			}
+		}
```
Comment (AroonSachin) [inline src/main/java/commonuseractions/CommonActionMethods.java:820]: Please remove lines from 655 to 820 then take a pull from develop which has these refactored methods for swipe actions.

**Review by AroonSachin — COMMENTED** [review]
There are changes to be done before merging the branch.

File: src/main/java/mobile_pageobjects/eribank/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+	@AndroidFindBy(id = "com.experitest.ExperiBank:id/loginButton")
+	@CacheLookup
+	WebElement sigin;
+
+	@AndroidFindBy(id = "com.experitest.ExperiBank:id/makePaymentButton")
+	@CacheLookup
+	WebElement makepayment;
+
+	@AndroidFindBy(xpath = "/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout[2]/android.widget.LinearLayout/android.webkit.WebView/android.webkit.WebView/android.view.View")
```
Comment (Vinoth-Velayutham) [inline src/main/java/mobile_pageobjects/eribank/LoginPage.java:34]: Need to check the xpath

**Review by Vinoth-Velayutham — APPROVED** [review]

**Pull request merged**